### PR TITLE
fix: add margin-top to footer #159

### LINKED ISSUE #159: Missing UI margin
### Description

I think there should be a margin below the add button.

Add margin top to footer, so every page it has a gap

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Bridgetamana**
## Fixes Issue

close #159 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Before
After

### CODE REVIEW

**Comment by eddiejaoude** [comment]
@Bridgetamana thank you, please can you add a before and after screenshot to the Pull Request description

**Comment by Bridgetamana** [comment]
Done

File: src/components/Footer.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const navigation = [
 
 export default function Footer() {
   return (
-    <footer className="bg-gray-800 mt-auto">
```
Comment (eddiejaoude) [inline src/components/Footer.js:35]: I think there are now 2 `mt-`, so the `mt-5` will override the `mt-auto`, can `mt-auto` be removed? Please test it again after the change

Comment (Bridgetamana) [reply src/components/Footer.js:35]: Thanks for pointing that out, I've removed the `mt-auto` class and everything looks good now.

Comment (eddiejaoude) [reply src/components/Footer.js:35]: ok great, thank you @Bridgetamana 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Review by Bashamega — APPROVED** [review]

**Comment by Bashamega** [comment]
closes #159

**Pull request merged**